OvernightPrintsSpecBindings/TestBase/Pages/Products/RemoveItemConfirmPopup.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using OpenQA.Selenium;
+
+namespace OvernightPrintsSpecBindings.TestBase.Pages.Products
+{
+	public class RemoveItemConfirmPopup
+	{
+		private readonly By _headerPageLocator = By.CssSelector("#remove-item-confirm > div > h3");
+		private readonly By _confirmButtonLocator = By.Id("remove-yes");
+		private readonly By _declineButtonLocator = By.Id("remove-no");
+
+		private HtmlElement HeaderPageElement;
+		private HtmlElement ConfirmButton;
+		private HtmlElement DeclineButton;
+
+		public RemoveItemConfirmPopup()
+		{
+			HeaderPageElement = new HtmlElement(_headerPageLocator);
+			ConfirmButton = new HtmlElement(_confirmButtonLocator);
+			ConfirmButton.WaitElementAppears();

```

Comment:
Same thing: it's better to put waits as separate step. Here you just initialize elements